src/modules/firebaseApp.ts

```diff
@@ -29,14 +29,14 @@ const dbFoodForum = ref(db, '/SNLApp/Forum/food/');
 const dbWineForum = ref(db, '/SNLApp/Forum/wine/');
 
 
-export function getBeerInDb(callback: (topic:Topic) => void) {
+export function getBeerInDb(callback: (topic: Topic) => void) {
```

Comment:
was there an error with the code because of the space?

Reply:
Hmm, I haven't changed that? Haha, that's weird that it says I did something on that line. 
Maybe I accidentally had the mouse there at one point and did something?! Anyway, it wasn't intentional so I don't know if it made a difference or not :angel:

Reply:
haha alright! was just wondering, maybe VS code automatically does it :) anyways, really good job!!